### DISCUSSION TITLE
pypi trove classifiers

### DISCUSSION THREAD

**Discussion by Stuermer** [python-poetry/poetry#7126] [Q&A] [answered]
Hi,
I'm not sure if this is a bug, if my expectations are wrong, or if the error is in my coding.
In the pyproject.toml of my package, I specified the python version do be:

```
[tool.poetry.dependencies]
python = ">=3.8,<3.10"
```

After uploading it to [PyPi](https://pypi.org/project/pyechelle/), the trove classifiers (correctly) showed up as:
![image](https://user-images.githubusercontent.com/6728233/204853777-942bc387-2e6c-4b7d-9e00-ac2c670b4a6d.png)

However, after changing the dependencies to 

```
[tool.poetry.dependencies]
python = ">=3.8,<3.11"
```
the classifiers didn't change and python 3.10 is not reported. 
My understanding was that these classifiers get extracted from the required python version automatically ? 
What am I missing?
Thanks in advance!

**Comment by radoering** [comment]
What's your poetry version?

**Reply by Stuermer** [reply]
That was poetry v1.1.7

**Comment by neersighted** [comment, marked as answer]
You are on a very obsolete version of Poetry; please update to 1.1.15 at the latest (but not it's unsupported; the 1.2.x branch is currently maintained and highly suggested).
